src/controllers/update/confirm.overseas.entity.details.controller.ts

```diff
@@ -51,6 +51,9 @@ export const post = (req: Request, res: Response, next: NextFunction) => {
       return res.redirect(config.UPDATE_TRUSTS_SUBMIT_BY_PAPER_URL);
     }
 
+    if (isRemoveJourney(req)) {
+      return res.redirect(`${config.PRESENTER_URL}${config.JOURNEY_REMOVE_QUERY_PARAM}`);
```

Comment:
`PRESENTER_URL` is the registration presenter page. We need to redirect to the update version of that page here - `OVERSEAS_ENTITY_PRESENTER_URL`.

Reply:
Updated to use correct presenter